libs/core/src/lib/form/inline-help-placement.type.ts

```diff
@@ -0,0 +1 @@
+export type InlineHelpPlacement = 'before' | 'after';
```

Comment:
We had a similar type in the inline-help module, but there are also `top` & `bottom` positions present, shouldn't we use that type instead?